### ISSUE TITLE
[BUG] Fresh install doesn't run

### ISSUE THREAD

**Issue by twchapman** [MontrealCorpusTools/Montreal-Forced-Aligner#655]
**Debugging checklist**

[x] Have you updated to latest MFA version?
[x] Have you tried rerunning the command with the `--clean` flag?

**Describe the issue**
I tried following the [Installation](https://montreal-forced-aligner.readthedocs.io/en/latest/installation.html#installation) instructions, which seemed to go smoothly, but even just trying to run `mfa` with no parameters errors out. 

I got this traceback and I'm really not sure what to do with it:
```
(aligner) D:\mfa2>mfa --clean
Traceback (most recent call last):
  File "C:\Users\t.chapman\AppData\Local\anaconda3\envs\aligner\Scripts\mfa-script.py", line 5, in <module>
    from montreal_forced_aligner.command_line.mfa import mfa_cli
  File "C:\Users\t.chapman\AppData\Local\anaconda3\envs\aligner\lib\site-packages\montreal_forced_aligner\__init__.py", line 6, in <module>
    import montreal_forced_aligner.command_line as command_line
  File "C:\Users\t.chapman\AppData\Local\anaconda3\envs\aligner\lib\site-packages\montreal_forced_aligner\command_line\__init__.py", line 12, in <module>
    from montreal_forced_aligner.command_line.diarize_speakers import diarize_speakers_cli
  File "C:\Users\t.chapman\AppData\Local\anaconda3\envs\aligner\lib\site-packages\montreal_forced_aligner\command_line\diarize_speakers.py", line 12, in <module>
    from montreal_forced_aligner.diarization.speaker_diarizer import SpeakerDiarizer
  File "C:\Users\t.chapman\AppData\Local\anaconda3\envs\aligner\lib\site-packages\montreal_forced_aligner\diarization\speaker_diarizer.py", line 59, in <module>
    from montreal_forced_aligner.diarization.multiprocessing import (
  File "C:\Users\t.chapman\AppData\Local\anaconda3\envs\aligner\lib\site-packages\montreal_forced_aligner\diarization\multiprocessing.py", line 15, in <module>
    import hdbscan
  File "C:\Users\t.chapman\AppData\Local\anaconda3\envs\aligner\lib\site-packages\hdbscan\__init__.py", line 1, in <module>
    from .hdbscan_ import HDBSCAN, hdbscan
  File "C:\Users\t.chapman\AppData\Local\anaconda3\envs\aligner\lib\site-packages\hdbscan\hdbscan_.py", line 40, in <module>
    FAST_METRICS = KDTree.valid_metrics + BallTree.valid_metrics + ["cosine", "arccos"]
TypeError: unsupported operand type(s) for +: 'builtin_function_or_method' and 'builtin_function_or_method'
```

**For Reproducing your issue**
Please fill out the following:

1. Corpus structure
   * What language is the corpus in? n/a
   * How many files/speakers? n/a
   * Are you using lab files or TextGrid files for input? n/a
2. Dictionary
   * Are you using a dictionary from MFA? If so, which one? n/a
   * If it's a custom dictionary, what is the phoneset? n/a
3. Acoustic model
   * If you're using an acoustic model, is it one download through MFA? If so, which one? n/a
   * If it's a model you've trained, what data was it trained on? n/a

**Log file**
n/a

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: Windows 10
 - Any other details about the setup (Cloud, Docker, etc): Using Anaconda on desktop

**Additional context**

**Comment by huangruizhe** [comment]
I got the same error with a fresh installation on Linux and python 3.10

**Comment by huangruizhe** [comment]
This issue happens elsewhere too due to the [latest released scikit-learn 1.3.0](https://github.com/scikit-learn/scikit-learn/releases), like [here](https://github.com/scikit-learn-contrib/hdbscan/issues/597) or [here](https://github.com/LibrePhotos/librephotos/pull/905).

As a workaround, I downgraded `scikit-learn` to a previous version:
```
pip uninstall scikit-learn
pip install scikit-learn==1.2.2
```
